src/redux/reducers/index.ts

```diff
@@ -2,11 +2,9 @@
 import { combineReducers } from "redux";
 
 import tweetsSlice from "./tweets";
-import { userSlice } from "./user";
+import userSlice from "./user";
```

Comment:
import tweetsReducer from "./tweets";
import userReducer from "./user";

at that place we have reducers not slice!